irods/query.py

```diff
@@ -233,6 +233,9 @@ def close(self):
         """
         self.limit(0).execute()
 
+    def _all(self):
+        return self.get_results()
```

Comment:
How is this different from `.all()`?

Reply:
Makes sense. So, `._all()` is guaranteed to return ALL rows regardless of any configuration, correct?

Reply:
Right

Reply:
> We should put that in a comment or docstring so we don't forget.

done